pymc3/variational/inference.py

```diff
@@ -749,42 +773,28 @@ def fit(n=10000, local_rv=None, method='advi', model=None,
     else:
         inf_kwargs = inf_kwargs.copy()
     if local_rv is not None:
-        inf_kwargs['local_rv'] = local_rv
+        inf_kwargs["local_rv"] = local_rv
     if random_seed is not None:
-        inf_kwargs['random_seed'] = random_seed
+        inf_kwargs["random_seed"] = random_seed
     if start is not None:
-        inf_kwargs['start'] = start
+        inf_kwargs["start"] = start
     if model is None:
         model = pm.modelcontext(model)
-    _select = dict(
-        advi=ADVI,
-        fullrank_advi=FullRankADVI,
-        svgd=SVGD,
-        asvgd=ASVGD,
-        nfvi=NFVI
-    )
+    _select = dict(advi=ADVI, fullrank_advi=FullRankADVI, svgd=SVGD, asvgd=ASVGD, nfvi=NFVI)
     if isinstance(method, str):
         method = method.lower()
-        if method.startswith('nfvi='):
+        if method.startswith("nfvi="):
             formula = method[5:]
-            inference = NFVI(
-                formula,
-                **inf_kwargs
-                )
+            inference = NFVI(formula, **inf_kwargs)
         elif method in _select:
 
-            inference = _select[method](
-                model=model,
-                **inf_kwargs
-            )
+            inference = _select[method](model=model, **inf_kwargs)
         else:
-            raise KeyError('method should be one of %s '
-                           'or Inference instance' %
-                           set(_select.keys()))
+            raise KeyError(
+                "method should be one of %s " "or Inference instance" % set(_select.keys())
```

Comment:
black did a bad job folding this, I think
```suggestion
                "method should be one of %s or Inference instance" % set(_select.keys())
```